### DISCUSSION TITLE
Not Verifying My Documents

### DISCUSSION THREAD

**Discussion by PerspicaciousGuy** [community/community#103386] [GitHub Education] [answered]
### Hi everyone,

I'm trying to apply for the GitHub student Developer Pack but the system won't let me do it. Whenever I upload my documents and click on "Process my application" it doesn't work and the page gets reload all the time and asks to upload the documents again.

Update: I guess they fixed the bug, now I got verified with the same docs.

**Comment by yassinebkr** [comment]
same +1

**Reply by User-23173** [reply]
Please help me to recover my account back normal

**Comment by mishmanners** [comment, marked as answer]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:

- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file is less than 150kB

**Reply by mishmanners** [reply]
I am glad you were able to resolve this 😄
